src/components/15Years/15YearsGlobe.vue

```diff
@@ -277,16 +278,12 @@ export default {
 
 .dot-callout {
 	transition: opacity 0.25s linear;
-	background: white;
-	width: calc(4 / 320 * 100vw);
-	height: calc(4 / 320 * 100vw);
+	background: $tomato;
+	border: 2px solid white;
 	border-radius: 50%;
 	box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.25);
-
-	@include breakpoint(large) {
-		width: 7px;
-		height: 7px;
-	}
+	width: 16px;
+	height: 16px;
```

Comment:
Can this be rems instead or does it have to be px for globekit?

Reply:
It can be rems. I used px because everything else in this file was px. Will do a follow-up PR with px2rem conversions for this component and 15YearsHeader.vue